pkg/state/state_ofed.go

```diff
@@ -248,9 +262,9 @@ func (s *stateOFED) getManifestObjects(
 	}
 
 	additionalVolMounts := additionalVolumeMounts{}
+	osname := attrs[0].Attributes[nodeinfo.AttrTypeOSName]
 	// set any custom ssl key/certificate configuration provided
```

Comment:
any chance to move logic related to additional volume mounts to a separate method ?

like `getAdditionalVolumeMouts` or `handleAdditionalVolumeMounts` ?